types/react-table/index.d.ts

```diff
@@ -13,9 +9,9 @@ export type ReactTableFunction = (value?: any) => void;
 export type AccessorFunction<D = any> = (row: D) => any;
 export type Accessor<D = any> = string | string[] | AccessorFunction<D>;
 export type Aggregator = (values: any, rows: any) => any;
-export type TableCellRenderer = ((data: any, column: any) => React.ReactNode) | React.ReactNode;
+export type TableCellRenderer = ((data: CellInfo, column: any) => React.ReactNode) | React.ReactNode;
```

Comment:
Rename `data` to `cellInfo` for consistency?
```suggestion
export type TableCellRenderer = ((cellInfo: CellInfo, column: any) => React.ReactNode) | React.ReactNode;
```